### DISCUSSION TITLE
Pretty output in console

### DISCUSSION THREAD

**Discussion by rogemus** [vercel/turborepo#630] [Ideas]
Hi, 
recently I found a package similar to turborepo which is [ultra](https://github.com/folke/ultra-runner). Sadly this package is no longer maintained. But I think it has one advantage over turborepo which is its output in the console. It is very easy to see what script is currently running which is done etc. What do you think of implementing a similar feature into turborepo? 

![output](https://github.com/folke/ultra-runner/raw/master/assets/ultra-lint.png)

**Comment by leonardssh** [comment]
Pretty interesting concept, I think it would be nice to have something similar in Turborepo (not saying the current one doesn't do the job, especially that `>>> FULL TURBO` gradient) 😅

**Comment by HananoshikaYomaru** [comment]
# 🥵 when I am using turborepo

![CleanShot 2023-05-17 at 12 11 35](https://github.com/vercel/turbo/assets/43137033/f162252b-3ed8-43a3-bcb5-f63d1fd965f5)

# 😌 when I typecheck inside the workspace

![CleanShot 2023-05-17 at 12 12 25](https://github.com/vercel/turbo/assets/43137033/0a2e5b2c-d4ac-4d4c-bcee-dcace26bcca3)

**Reply by aagwali** [reply]
Hello,

For the particular case of type check running `tsc --noEmit --pretty` will give you back the nice look when using turborepo.

![image](https://github.com/vercel/turbo/assets/48925533/5c133297-ab3f-43af-8218-c51215905a77)

However I'm stuck with the same issue when it comes to eslint for instance

**Comment by anthonyshew** [comment]
Turborepo 2.0 has a new terminal UI that you might like!
